ui/pages/permissions-connect/snaps/snap-result/snap-result.js

```diff
@@ -168,9 +175,7 @@ export default function SnapResult({
         className="snap-result__footer"
         alignItems={AlignItems.center}
         flexDirection={FlexDirection.Column}
-        style={{
```

Comment:
Is it intentional to drop the box shadow for the "scroll area overlay"? 🤔

Reply:
Yes, we're dropping the visual container for the footer